packages/mdc-select/index.js

```diff
@@ -287,7 +290,9 @@ class MDCSelect extends MDCComponent {
     const targetClientRect = evt.target.getBoundingClientRect();
     const xCoordinate = evt.clientX;
     const normalizedX = xCoordinate - targetClientRect.left;
-    this.lineRipple_.setRippleCenter(normalizedX);
+    if (this.lineRipple_) {
```

Comment:
This would never not be true since we're not hooking up the listener to this method if line ripple doesn't exist, right?